client/core/core_test.go

```diff
@@ -1482,13 +1497,83 @@ func TestLogin(t *testing.T) {
 	}
 
 	// Success again.
-	rig.queueConnect()
+	rig = newTestRig()
+	tCore = rig.core
+	rig.acct.markFeePaid()
+	rig.queueConnect(nil)
 	_, err = tCore.Login(tPW)
 	if err != nil || !rig.acct.authed() {
 		t.Fatalf("final Login error: %v", err)
 	}
 }
 
+func TestLoginAccountNotFoundError(t *testing.T) {
+	rig := newTestRig()
+	tCore := rig.core
+	rig.acct.markFeePaid()
+
+	expectedErrorMessage := "test account not found error"
+	accountNotFoundError := msgjson.NewError(msgjson.AccountNotFoundError, expectedErrorMessage)
+	rig.queueConnect(accountNotFoundError)
+
+	wallet, _ := newTWallet(tDCR.ID)
+	tCore.wallets[tDCR.ID] = wallet
+	rig.queueConnect(nil)
+
+	result, err := tCore.Login(tPW)
+	if err != nil {
+		t.Fatalf("unexpected Login error: %v", err)
+	}
+	for _, dexStat := range result.DEXes {
+		if dexStat.Authed || dexStat.AuthErr == "" || !strings.Contains(dexStat.AuthErr, expectedErrorMessage) {
+			t.Fatalf("expected account not found error")
+		}
+	}
+}
+
+func TestInitializeDEXConnectionsSuccess(t *testing.T) {
```

Comment:
Thanks for adding the `dexStats` checks here. What do you think about merging this test with `TestLogin`?

Reply:
I put this into it's own test so that it's easy to test `InitializeDEXConnections()`. I wanted to test only `InitializeDEXConnections()` and if it was part of the `Login()` test I had to sift though the multiple test results to see my specific test result. 

Also if the test stands alone I can put a debug breakpoint on the thing that I'm debugging without having to put an initial breakpoint in test code and then placing the breakpoint in the code.

I realized I was breaking from the pattern that is prevalent in the existing tests. I am interested in your opinions on this.